packages/components/src/Combobox/Combobox.test.tsx

```diff
@@ -1,530 +1,339 @@
-      render(
-        <Combobox>
-          <Combobox.TriggerButton label="Button" />
-          <Combobox.Content
-            options={[]}
-            onSelect={jest.fn()}
-            selected={[]}
-          ></Combobox.Content>
-        </Combobox>,
-      );
-    }).not.toThrow();
-  });
+const MENU_TEST_ID = "ATL-Combobox-Content";
+const OVERLAY_TEST_ID = "ATL-Combobox-Overlay";
 
-  it("renders without error when there is no TriggerButton or TriggerChip", () => {
-    expect(() => {
-      render(
-        <Combobox>
-          <Combobox.Content
-            options={[]}
-            onSelect={jest.fn()}
-            selected={[]}
-          ></Combobox.Content>
-        </Combobox>,
-      );
-    }).not.toThrow();
-  });
-
-  it("renders without error when there is a ComboboxActivator", () => {
-    expect(() => {
-      render(
-        <Combobox>
-          <Combobox.Activator>
-            <Chip variation="subtle" label="Teammates" />
-          </Combobox.Activator>
-          <Combobox.Content
-            options={[]}
-            onSelect={jest.fn()}
-            selected={[]}
-          ></Combobox.Content>
-        </Combobox>,
-      );
-    }).not.toThrow();
-  });
-
-  it("throws an error if there are multiple of the same Trigger element", () => {
-    expect.assertions(1);
-    let error;
-
-    try {
-      render(
-        <Combobox>
-          <Combobox.TriggerButton label="Button" />
-          <Combobox.TriggerButton label="Button Again" />
-          <Combobox.Content
-            options={[]}
-            onSelect={jest.fn()}
-            selected={[]}
-          ></Combobox.Content>
-        </Combobox>,
-      );
-    } catch (e) {
-      error = e as Error;
-    } finally {
-      expect(error?.message).toBe(COMBOBOX_TRIGGER_COUNT_ERROR_MESSAGE);
-    }
-  });
-
-  it("throws an error if there are multiple of various Trigger elements", () => {
-    expect.assertions(1);
-    let error;
-
-    try {
-      render(
-        <Combobox>
-          <Combobox.TriggerButton label="Button" />
-          <Combobox.TriggerChip label="Chippy Chip" />
-          <Combobox.Content
-            options={[]}
-            onSelect={jest.fn()}
-            selected={[]}
-          ></Combobox.Content>
-        </Combobox>,
-      );
-    } catch (e) {
-      error = e as Error;
-    } finally {
-      expect(error?.message).toBe(COMBOBOX_TRIGGER_COUNT_ERROR_MESSAGE);
-    }
-  });
-
-  it("throws an error if there is a Trigger element and a ComboboxActivator", () => {
-    const component = () =>
-      render(
-        <Combobox>
-          <Combobox.TriggerButton label="Button" />
-          <Combobox.Activator>
-            <Chip variation="subtle" label="Teammates" />
-          </Combobox.Activator>
-          <Combobox.Content
-            options={[]}
-            onSelect={jest.fn()}
-            selected={[]}
-          ></Combobox.Content>
-        </Combobox>,
-      );
+const activatorLabel = "Select a Baggins";
```

Comment:
so this does work, but what do you think about instead of having a static heading + getByText, we do `getByRole("combobox")` which will target the activator button in a more generic way

this will help with translations and all that sort of stuff down the road!